packages/sharing-smart-contract/test/docker-compose.yml

```
@@ -0,0 +1,35 @@
+services:
+  bellecour-fork-voucher-deployment:
+    restart: 'no'
+    # https://github.com/iExecBlockchainComputing/voucher-deployer
+    image: pierreiexec/voucher-deployer
+    platform: linux/amd64
+    environment:
+      RPC_URL: http://host.docker.internal:8545
+      SKIP_SUBGRAPH: 'true'
+    volumes:
+      - voucher-deployer:/app/out

```

`host.docker.internal` does not work on linux, maybe this can work for both linux and mac.

```suggestion
  bellecour-fork-voucher-deployment:
    restart: 'no'
    # https://github.com/iExecBlockchainComputing/voucher-deployer
    image: pierreiexec/voucher-deployer
    platform: linux/amd64
    environment:
      RPC_URL: http://localhost:8545
      SKIP_SUBGRAPH: 'true'
    volumes:
      - voucher-deployer:/app/out
    network_mode: 'host'
```